### DISCUSSION TITLE
PI 12 Iteration 3

### DISCUSSION THREAD

**Discussion by mkanal** [eclipse-tractusx/item-relationship-service#432] [General]
GOAL:  Data Sovereignty 

## Minister Security
@ds-psosnowski 
Timebox: 2h
- [ ] Check Veracode for findings
- [ ] Check Invict for findings
- [ ] run and check Dashtool IP

## Release Champion: 
@ds-jhartmann 
Timebox: 2h

- [ ] Has the Release been conducted (Release is being skipped!)
- [ ] Check for Dependabots PR's on Tractus-X
- [ ] Update [COMPATIBILITY_MATRIX](https://github.com/catenax-ng/tx-item-relationship-service/blob/main/COMPATIBILITY_MATRIX.md) with new Version

## Docu-Guru:
@ds-ext-kmassalski 
Timebox: 2h 
- [ ] Focused on Documentation of Tickets

## Testmanager
@mkanal / @ds-alexander-bulgakov 
- [x] Update [Release 24.03 ---- Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.03+----+Content+Overview)
- [ ] Update Test Status - PI12 [Test Status - PI12](https://confluence.catena-x.net/pages/viewpage.action?pageId=114072227)
- [ ] Update [Integration Test - PI12 - Release 2024.05](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI12+-+Release+2024.05)

## Firefighter
@dsmf 
- [ ] Maintenance of our community
- [ ] Response to Github issues https://github.com/eclipse-tractusx/item-relationship-service/issues
- [ ]  To be defined, eg: Onboarding of new colleague, quality-gates, escalation topics
- [ ] Communicating and discussing with external contributors in Eclipse Matrix Chat

**Comment by ds-alexander-bulgakov** [comment]
- [x] Update Test Status - PI12 [Test Status - PI12](https://confluence.catena-x.net/pages/viewpage.action?pageId=114072227)
